dist/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+    <!-- Link to Roboto and Inter Font-->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+
+    <!-- Link to the Font-Awesome Font -->
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" />
+
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous" />
+    <link rel="stylesheet" href="style.css" />
+    <!-- JavaScript Bundle with Popper -->
+
+    <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="container-fluid">
+        <h5 class="col-12 text-center py-2">A web page</h5>
+        <div class="row mx-5 headerList">
+            <i class="col-1 fa fa-arrow-left"></i>
+            <i class="col-1 fa fa-arrow-right"></i>
+            <i class="col-1 fa fa-times"></i>
+            <i class="col-1 fa-sharp fa fa-arrow-up"></i>
+            <input class="col-6 search" type="text" placeholder="https://" />
+            <i class="col-1 mx-5 fa fa-search"></i>
+        </div>
+    </header>
```

Comment:
- [ ] Kindly remove this unnecessary header as we would not be using it. The arrows, URL inputs, and search icon you see on the wireframe are just sketches of a browser. They are not part of the wireframe.

  ![nea](https://user-images.githubusercontent.com/54848363/187899167-5aaafe6c-d887-46b9-bdec-c14ca4087b9b.jpg)